### PR TITLE
CP-49934-Remove python2.7 test coverage.

### PR DESCRIPTION
**Pull request by ashwin9390**
Main task : CP-49100- Move all scripts from scripts folder to python3 folder and modify Makefile to include these changes.

Subtask: CP-49934 -Remove python2.7 test upload coverage as we don't require this anymore for python3 code.

 Signed-off-by: Ashwinh <ashwin.h@cloud.com>

@psafont @liulinC @stephenchengCloud

### CODE REVIEW

**Review by stephenchengCloud — APPROVED** [review]

**Review by liulinC — APPROVED** [review]

File: .github/workflows/other.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["2.7", "3.11"]
+        python-version: [ "3.11"]
```
Comment (psafont) [inline .github/workflows/other.yml:21]:
```suggestion
      python-version: "3.11"
```

File: .github/workflows/other.yml [hunk 2]
```diff
@@ -45,14 +45,6 @@ jobs:
...
           SKIP: no-commit-to-branch
 
-      - name: Run Pytest for python 2 and get code coverage for Codecov
-        if: ${{ matrix.python-version == '2.7' }}
-        run: >
-          pip install enum future mock pytest-coverage pytest-mock &&
-          pytest
-          --cov=scripts scripts --cov-fail-under 45 -vv -rA
-          --cov-report term-missing
-          --cov-report xml:.git/coverage${{matrix.python-version}}.xml
```
Comment (psafont) [inline .github/workflows/other.yml:55]: Please remove line 42 and other references to python 2 in this workflow

**Review by psafont — CHANGES_REQUESTED** [review]
Don't remove the tests just yet, instead change the coverage upload function to not be run when using python2:

```
      - name: Upload Python ${{matrix.python-version}} coverage report to Codecov
        if: ${{ matrix.python-version != '2.7' }}
        uses: codecov/codecov-action@v3
        with:
          directory: .git
          files: coverage${{matrix.python-version}}.xml
          env_vars: OS,PYTHON
          fail_ci_if_error: false
          flags: python${{matrix.python-version}}
          name: coverage${{matrix.python-version}}
          verbose: true
```

File: .github/workflows/other.yml [hunk 3]
```diff
@@ -45,16 +45,9 @@ jobs:
         env:
           SKIP: no-commit-to-branch
 
-      - name: Run Pytest for python 2 and get code coverage for Codecov
```
Comment (psafont) [inline .github/workflows/other.yml:48]: Restore all the removed lines, please

**Review by psafont — APPROVED** [review]

**Pull request merged**